docs/API.md

```diff
@@ -433,55 +428,6 @@ class App extends React.Component {
 }
 ```
 
-### Named Components
```

Comment:
On second thoughts you're right, thanks. I've expanded slightly to try and clarify further.

Would you rather I squashed these commits and resubmitted the PR?